zcash_primitives/src/consensus.rs

```diff
@@ -42,7 +42,7 @@ impl Parameters for TestNetwork {
             NetworkUpgrade::Sapling => Some(280_000),
             NetworkUpgrade::Blossom => Some(584_000),
             NetworkUpgrade::Heartwood => Some(903_800),
-            NetworkUpgrade::Canopy => None,
+            NetworkUpgrade::Canopy => Some(1_028_500),
```

Comment:
This matches the Canopy testnet activation height: https://github.com/zcash/zcash/blob/f39e4dbaf107aade8b9227c8b76acca711efc9a8/src/chainparams.cpp#L355